protostar/commands/install/install_command.py

```diff
@@ -114,3 +117,26 @@ def install(
                 repo_dir=self._project_root_path,
                 libs_dir=libs_path,
             )
+
+    def suggest_adding_cairo_path(self, libs_path, package_name):
+        protostar_toml = "protostar.toml"
+        shared_command_configs_key = "protostar.shared_command_configs"
+
+        data = tomli.loads(
+            (self._project_root_path / protostar_toml).read_text(encoding="utf-8")
+        )
+        shared_command_configs = data.get(shared_command_configs_key)
+        if not shared_command_configs:
+            shared_command_configs = {}
+        shared_command_configs["cairo_path"] = (
+            ["..."] if shared_command_configs.get("cairo_path") else []
+        )
+        shared_command_configs["cairo_path"].append(f"{libs_path}/{package_name}/src")
+
+        self._logger.info(
+            "You may want to add your new library's path to the 'cairo_path' section of your "
+            f"`{protostar_toml}` file"
+        )
+        self._logger.info(
+            f"\n[{shared_command_configs_key}]\n{tomli_w.dumps(shared_command_configs)}"
+        )
```

Comment:
[Design] I'd move some code to the Configuration File module e.g. `ConfigurationFile::create_appending_cairo_path_suggestion`.

Reply:
[move suggesting function to the ConfigurationFile](https://github.com/software-mansion/protostar/pull/981/commits/2bd3099efe10c13ed4982fd6a56509ebd4288d03)

Reply:
Sorry I didn't see this